src/lib/holocene/data-encoder-settings.svelte

```diff
@@ -90,15 +90,14 @@
           href="https://docs.temporal.io/dataconversion#codec-server"
           class="text-blue-700 underline decoration-blue-700">Codec Server</a
         >
-        decodes your data. A Codec
-        Server endpoint can be set at the {namespaceOrCluster} level, or locally
-        in your browser.
+        decodes your data. A Codec Server endpoint can be set at the {namespaceOrCluster}
+        level, or locally in your browser.
       </p>
       <Accordion
         data-testid="override-accordion"
         title={override
-          ? `Uses my browser setting and ignores ${namespaceOrCluster}-level settings.`
-          : `Uses ${namespaceOrCluster}-level settings, where available.`}
+          ? `Use my browser setting and ignores ${namespaceOrCluster}-level setting.`
```

Comment:
```suggestion
          ? `Use my browser setting and ignore ${namespaceOrCluster}-level setting.`
```

Reply:
or "Use my browser setting. This ignores the ${namespaceOrCluster}-level setting."